### PR TITLE
[telnetpp] Add missing <ostream> for Visual Studio 2019 16.6

### PR DESCRIPTION
**Pull request by BillyONeal**
Previously submitted as https://github.com/KazDragon/telnetpp/pull/215

### CODE REVIEW

File: ports/telnetpp/github-215.patch [hunk 1]
```diff
@@ -0,0 +1,55 @@
+From df53cf93d35f74db22eaadcac87bd0c88ea87f5d Mon Sep 17 00:00:00 2001
+From: Cheney-Wang <v-xincwa@microsoft.com>
+Date: Wed, 12 Feb 2020 04:12:46 -0800
+Subject: [PATCH 1/2] Include <ostream> in variable.hpp
```
Comment (PhoebeHui) [inline ports/telnetpp/github-215.patch:4]:
```suggestion
```

File: ports/telnetpp/github-215.patch [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ #include <string>
++#include <ostream>
+ 
+ namespace telnetpp { namespace options { namespace msdp {
+ 
+
+From ea7477eba057fec5ca8178eaacd5b11c5745a9e7 Mon Sep 17 00:00:00 2001
+From: Cheney-Wang <v-xincwa@microsoft.com>
+Date: Thu, 20 Feb 2020 04:17:39 -0800
+Subject: [PATCH 2/2] Include <ostream> in variable.cpp
```
Comment (PhoebeHui) [inline ports/telnetpp/github-215.patch:26]:
```suggestion
```

**Review by PhoebeHui — CHANGES_REQUESTED** [review]
Remove the comments for making the patch file smaller.

**Comment by BillyONeal** [comment]
> Remove the comments for making the patch file smaller.

I put the comments there to make it identical to the GitHub PR commit so that it's clear once we take a version with that commit integrated that the patch can be removed.

**Comment by BillyONeal** [comment]
@PhoebeHui Looks like you removed the bits you didn't like yourself but you're still marked as requesting changes. Is there anything else you're looking for?

Thanks!

**Review by PhoebeHui — APPROVED** [review]
LGTM, thanks Billy!

**Pull request merged**